client/app/components/show-project/actions-list/ulurp-number-link.js

```diff
@@ -0,0 +1,28 @@
+import Component from '@ember/component';
+import ENV from 'labs-zap-search/config/environment';
+
+/**
+ * @param {Action Object} action
+ */
+export default class UlurpNumberLinkComponent extends Component {
+  host = ENV.host;
+
+  ACTION_STATUSCODE_ACTIVE = 1;
+
+  get serverRelativeUrl () {
+    const serverRelativeFolderUrl = this.action.dcpSpabsoluteurl.replace('https://nyco365.sharepoint.com', '');
```

Comment:
do UAT2 and prod have the same sharepoint folders?

Reply:
They both have the "nyco364.sharepoint.com" domain, but differ in the "environment" segment of the document URL